source/platforms/nvidia-ai-workbench.md

```diff
@@ -0,0 +1,63 @@
+# NVIDIA AI Workbench
+
+[NVIDIA AI Workbench](https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/workbench/) is a developer toolkit for data science, machine learning, and AI project development that lets you develop on your laptop/workstation and then move workloads to scalable GPU resources in a data center or the cloud. AI Workbench is free, you can install it in minutes on your local or remote computers, and it provides both a desktop application and a command-line interface (CLI).
+
+## Installation
+
+You can install AI Workbench locally, or on a remote computer that you have SSH access to.
+
+Follow the [AI Workbench installation](https://docs.nvidia.com/ai-workbench/user-guide/latest/installation/overview.html) documentation for instructions on installing on different operating systems.
+
+```{admonition} Thick client
+:class: note
+
+If you are working on a system that has an NVIDIA GPU you can use the AI Workbench installed to set up and configure Docker if you don't have it installed already. Then you can run notebooks in Python environments with access to your GPU.
+
+To use AI Workbench in this way you only need to install the desktop application.
```

Comment:
If we mention to install the desktop app above as I suggest, we don't need this. Feel free to ignore this change though.
```suggestion
```